Process.h

```diff
@@ -209,6 +209,9 @@ typedef struct ProcessFieldData_ {
 
    /* Whether the column width is dynamically adjusted (the minimum width is determined by the title length) */
    bool autoWidth;
+
+   /* Whether the title of a column with dynamically adjusted width is right aligned (default is left aligned) */
+   bool autoTitleRightAlign;
```

Comment:
Is there anything automatic about the alignment?  (i.e. could the name be simplified to just 'titleRightAlign'?)

Reply:
That `auto` refers to the automatic resize of certain columns.